README.md

```diff
@@ -219,20 +216,25 @@ There may be cases where the _idem-api_ module is able to request the website's
 
 | CredentialSubject   | Mnemonic     | Standard            | Description            |
 | ------------------- | ------------ | ------------------- | ---------------------- |
-| AdultCredential     | eighteenplus | 18 Plus             | 18 Plus                |
-| BirthCredential     | dob          | YYYY-MM-DD ISO 8601 | Users date of birth    |
-| NameCredential      | fullname     | Firstname Lastname  | Users full name        |
-| EmailCredential     | email        | Email               | Users email address    |
-| MobileCredential    | mobilenumber | Mobile Number       | Users mobile number    |
-| AddressCredential   | address      | Physical Address    | Users physical address |
-| BirthYearCredential | birthyear    | YYYY ISO 8601       | Users year of birth    |
+| AdultCredential     | eighteenplus | 18 Plus                             | 18 Plus                |
+| BirthCredential     | dob          | YYYY-MM-DD ISO 8601                 | Users date of birth    |
+| NameCredential      | fullname     | given,name, family name, middle name| Users full name        |
```

Comment:
Not sure about this little bit here "given,name, family name, middle name" does not need the "," between "given,name"